### PR TITLE
CVE-2017-7668

### PR DESCRIPTION
**Pull request by s-newman**
Round 1 initial submission.

### CODE REVIEW

File: cves/CVE-2017-7668.yml [hunk 1]
```diff
@@ -47,8 +47,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
-
+description: |
```
Comment (rmnill202) [inline cves/CVE-2017-7668.yml:50]: Great description! I might change the end where you say "would continue to read past the end of the string in a buffer overrun" to something like "would continue to read past the end of the string, **resulting in a** buffer overrun"

**Comment by rmnill202** [comment]
+2 upvotes

File: cves/CVE-2017-7668.yml (hunk 1 above)
Comment (s-newman) [reply cves/CVE-2017-7668.yml:50]: Ah, good point, I'll fix this. Thanks!

Comment (s-newman) [reply cves/CVE-2017-7668.yml:50]: Updated.

**Comment by sfm2686** [comment]
+1 upvote, I think you did a good job overall describing the CVE and answering the questions!

**Pull request merged**